### PR TITLE
Docs: Jon's updates to the poly shape tool page

### PR DESCRIPTION
**Pull request by suearkinunity**
### Purpose of this PR

Update to the poly shape tool page

### Links

**Jira:** DOCATT-5834

### Comments to Reviewers

@JonMUnity I left a comment right at the end about a note I don't understand.

### CODE REVIEW

**Comment by unity-cla-assistant** [comment]
[CLA assistant check](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.probuilder?pullRequest=537) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.probuilder?pullRequest=537) before we can accept your contribution.<br/><hr/>**Sue Arkin** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.cds.internal.unity3d.com/check/Unity-Technologies/com.unity.probuilder?pullRequest=537) it.</sub>

File: Documentation~/workflow-create-polyshape.md [hunk 1]
```diff
@@ -1,34 +1,29 @@
...
-	![Extruding the 2D shape into a 3D Mesh](images/PolyShape_Draw2.png)
-
-5. Move the mouse up and down to set the Mesh height. This extrudes a 3D Mesh from the 2D shape you defined with your control points.
-
-6. Click to exit create mode. Your new Mesh is now [in editing mode](polyshape.md) so you can continue to modify it.
-
-	![Finished Poly Shape](images/PolyShape_Draw3.png)
+> **Note**: When you modify control points or settings, you lose any changes you made in editing the mesh.
 
+[[Jon: I don't understand this note. What is the timeline for losing changes? As in, does it only happen if *while I'm editing* I also change control points or settings? How do I change control points or settings? Whose settings?]]
```
Comment (JonMUnity) [inline Documentation~/workflow-create-polyshape.md:29]: Good question. I think it only happens when you're editing. @modrimkus is this true?

File: Documentation~/workflow-create-polyshape.md [hunk 2]
```diff
@@ -1,34 +1,29 @@
...
 
-> **Note**: When you modify control points or settings, you lose any changes you made in editing the Mesh.
+![A Poly Shape is a 3D shape extruded from any 2D polygon](images/PolyShape_HeaderImage.png)
 
-![A Poly Shape is a 3-dimensional shape extruded from any 2-dimensional polygon](images/PolyShape_HeaderImage.png)
+To create a custom mesh:
 
-To define a custom Mesh:
+1. Use one of the following:
+    * In the **Scene** view > **Tools** overlay, select **Create PolyShape**.
```
Comment (JonMUnity) [inline Documentation~/workflow-create-polyshape.md:10]:
```suggestion
    * In the Scene view, in the **Tools** overlay, select **Create PolyShape**.
```

File: Documentation~/workflow-create-polyshape.md [hunk 3]
```diff
@@ -1,34 +1,29 @@
...
+    * In the main menu, go to **Tools** > **ProBuilder** > **Editors** > **New Poly Shape**.
```
Comment (JonMUnity) [inline Documentation~/workflow-create-polyshape.md:11]:
```suggestion
    * In the main menu, go to **Tools** > **ProBuilder** > **Editors** > **New PolyShape**.
```

File: Documentation~/workflow-create-polyshape.md [hunk 4]
```diff
@@ -1,34 +1,29 @@
...
+1. To create the outer bounds of your mesh, click in the **Scene** view to create control points.
```
Comment (JonMUnity) [inline Documentation~/workflow-create-polyshape.md:12]:
```suggestion
1. To create the outer bounds of your mesh, click in the Scene view to create control points.
```

File: Documentation~/workflow-create-polyshape.md (hunk 1 above)
Comment (modrimkus-unity) [reply Documentation~/workflow-create-polyshape.md:29]: @suearkinunity @JonMUnity The note is trying to say that regular mesh editing vs working procedurally with shapes don't really mix that well. You can't do:
1) procedural shape edits
2) direct vert/edge/face manipulation on top
3) procedural shape edits on top

You can do the first two steps and continue to say in low level manipulations but you can't return to the high/procedural level easily/without side-effects. 

When you try to do the 3rd step, prior to PB6, we'd remove the direct edits (2) and automatically put you back in base shape editing (1) and in PB6 there's now a warning shown and you have to "Reset shape" explicitly to go back to procedural editing.

https://github.com/user-attachments/assets/d7eadd8b-1fb6-457d-a3d9-216f5c7b53dc

File: Documentation~/workflow-create-polyshape.md (hunk 2 above)
Comment (suearkinunity) [reply Documentation~/workflow-create-polyshape.md:10]: Style guide says **Scene** should be in bold

Comment (JonMUnity) [reply Documentation~/workflow-create-polyshape.md]: My bad. Disregard then.

File: Documentation~/workflow-create-polyshape.md [hunk 5]
```diff
@@ -1,34 +1,29 @@
...
-4. When you finish placing points, use either **Enter** (Windows), **Return** (macOS), or **Spacebar** to finalize the shape.
-
-	![Extruding the 2D shape into a 3D Mesh](images/PolyShape_Draw2.png)
-
-5. Move the mouse up and down to set the Mesh height. This extrudes a 3D Mesh from the 2D shape you defined with your control points.
-
-6. Click to exit create mode. Your new Mesh is now [in editing mode](polyshape.md) so you can continue to modify it.
-
-	![Finished Poly Shape](images/PolyShape_Draw3.png)
+> **Note**: When you modify control points or settings, you lose any changes you made in editing the mesh.
```
Comment (suearkinunity) [inline Documentation~/workflow-create-polyshape.md:27]:
```suggestion
> **Note**: If you apply procedural edits to the mesh, any manual changes are overwritten.
```

File: Documentation~/workflow-create-polyshape.md [hunk 6]
```diff
@@ -1,34 +1,29 @@
...
 
+[[Jon: I don't understand this note. What is the timeline for losing changes? As in, does it only happen if *while I'm editing* I also change control points or settings? How do I change control points or settings? Whose settings?]]
```
Comment (suearkinunity) [inline Documentation~/workflow-create-polyshape.md:29]:
```suggestion
```

File: Documentation~/workflow-create-polyshape.md (hunk 5 above)
Comment (suearkinunity) [reply Documentation~/workflow-create-polyshape.md:27]: Okay, trying to reword the note. 
@JonMUnity what do you think? Oversimplified?

**Review by JonMUnity — APPROVED** [review]

**Comment by suearkinunity** [comment]
@modrimkus-unity please can you merge this?

**Pull request merged**